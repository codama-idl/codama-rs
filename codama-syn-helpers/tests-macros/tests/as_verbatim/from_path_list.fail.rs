use codama_syn_helpers_test_macros::as_verbatim;

#[as_verbatim(foo(1, 2, 3))]
pub struct Test;

#[as_verbatim(foo = (1, 2, 3))]
pub struct TestWithEq;

fn main() {}
