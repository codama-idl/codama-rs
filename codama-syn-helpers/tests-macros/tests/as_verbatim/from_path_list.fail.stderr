error: expected verbatim
 --> tests/as_verbatim/from_path_list.fail.rs:3:15
  |
3 | #[as_verbatim(foo(1, 2, 3))]
  |               ^^^^^^^^^^^^

error: expected verbatim
 --> tests/as_verbatim/from_path_list.fail.rs:6:15
  |
6 | #[as_verbatim(foo = (1, 2, 3))]
  |               ^^^^^^^^^^^^^^^
